apps/storefront/components/MdxContent/MdxContent.module.css

```diff
@@ -124,6 +124,10 @@
     border-radius: var(--ds-border-radius-md);
   }
 
+  & pre code {
+    background-color: unset;
```

Comment:
This looks like it's doing something other than max-width, is the commit message wrong?

Reply:
yeah this was another small bug I noticed. Can add to the description